### DISCUSSION TITLE
Can I use i18next for web page translations?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [i18next/i18next#2071] [Q&A]
Newbie here ...
I looked at the docs and seems like i18next is oriented towards SPAs and JS frameworks (React, Vue). I need something fast and weight, to translate plain web pages in the browser, on-the-fly, basically replacing the inner html with keys from the element attributes.
There are these two JS Libraries that seem to do exactly what I need, but are far less popular:
- [TranslatorJS](https://github.com/EOussama/translator.js)
- [Simple Translator](https://github.com/andreasremdt/simple-translator)

... so I wanted to see if i18next can do similar job, and could not find anything in the docs. Maybe there is a plug-in/extension  that does what the other two JS libs do?

**Comment by jamuhl** [comment]
https://github.com/i18next/i18nextify is what you're looking for

there is also - https://github.com/locize/locizify (connects to our TMS with all benefits of having a professional solution)

**Reply by jamuhl** [reply]
or https://github.com/mthh/loc-i18next if you need more control
